src/Symfony/Component/VarDumper/Tests/Dumper/functions/dump_without_args.phpt

```diff
@@ -10,7 +10,7 @@ while (!file_exists($vendor.'/vendor')) {
 }
 require $vendor.'/vendor/autoload.php';
 
-dump();
```

Comment:
by adding a destructor to the options class, we could (and should IMHO) keep dump() as is